### ISSUE TITLE
Brodie bug: The icons in the lower right are overwhelming and unclear 

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#3098]
The icons in the lower right are overwhelming and unclear of their function at a glance. Maybe add text under them (icon + text is usually stronger for initial UI). That may add clutter but at this point I'd rather have clarity over anything else.

**Comment by MarjieAVolk** [comment]
Yeah, this goes along with reducing the number of buttons down there too.  👍

**Comment by elfl0rd** [comment]
@D3nnis3n as i see this target reworked...

**Comment by D3nnis3n** [comment]
Don't see any change on the buttons - still no text on them.
